src/views/Package/PackageDocs.tsx

```diff
@@ -124,7 +77,14 @@ export const PackageDocs: FunctionComponent<PackageDocsProps> = ({
           },
         }}
       >
-        {markdown}
+        <Switch>
+          <Route exact path={path}>
+            <PackageReadme />
+          </Route>
+          <Route exact path={`${path}/api/:typeId`}>
+            <PackageTypeDocs />
+          </Route>
```

Comment:
Would using `LazyRoute` here improve performance at all?

Reply:
Very little, basically saves including PackageTypeDocs and/or PackageReadme components if youre on one or the other, which are both pretty tiny.